### PR TITLE
Refactor functions

### PR DESCRIPTION
**Pull request by Talha-Hanif5666**
### **_Highlights_**
 **Add ES6 syntax.
 Change functions to an arrow function.
 Use npm as a software package ages system.
 Import modules.**

### CODE REVIEW

File: modules/content.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      bookArray.forEach((element, index) => {
+        const displayTitle = document.createElement('p');
+        const displayAuth = document.createElement('p');
+        const deletebtn = document.createElement('div');
+        const container = document.createElement('div');
+        const words = document.createElement('div');
+        // set attributes
+        displayTitle.innerText = `"${element.title}" by`;
+        displayAuth.innerText = element.author;
+        deletebtn.innerHTML = `<button class="btn borders" onclick='deleteItem(${index})'>Remove</button>`;
```
Comment (DammyShittu) [inline modules/content.js:55]: - [ ] When I click to delete a book the book is not removed from the local storage or the page and I get this error. Kindly fix that.

  fkffo

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      <p>Do have any questions or you just want to say "Hello"?</p>
+      <p>You can reach out to us!</p>
+      <ul>
+        <li>Our email mail@gmail.com</li>
+        <li>Our phone no. 0998437453</li>
+        <li>Our address: xyz</li>
+      </ul>
+    </section>
+  </div>
+  <footer id="foots">Copyright &copy; Muhammad Talha Hanif, Amaka Laura & Christian Ani</footer>
```
Comment (DammyShittu) [inline index.html:52]: - **[OPTIONAL]** I think it would be nice fix the footer to the bottom of the page.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Talha-Hanif5666,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#   *Status: Approved* :tada: :heavy_check_mark: 

Hi @Talha-Hanif5666, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark:



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**